StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerHelpers.swift

```diff
@@ -8,11 +8,16 @@
 import UIKit
 
 final class AccountPickerHelpers {
-    static func rowTitles(forAccount account: FinancialConnectionsPartnerAccount) -> (
+    static func rowTitles(
+        forAccount account: FinancialConnectionsPartnerAccount,
+        // caption, for networked accounts, _will_ hide account numbers, so we should show account numbers in the title
```

Comment:
is `_xxx_` going to generate specific fonts in doc? 

Also is intended `_will_` here and `_hide` at line 18?

Reply:
haha this is simply my 'voice' coming out in text comments; in this case, underscore is often markdown syntax for emphasis/italics. there's not much more to it than me emphasizing parts. I can remove it if needed

Reply:
I removed to simplify the discussion haha

Reply:
oh in android you could actually use some syntax in comments so they are formatted when generated to doc, wish swift has this too